_posts/2024-04-01-pyos-newsletter-april-2024.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: April 2024"
+excerpt: "New contributors, a new package, and a few new social campaigns, all to keep you connected with the pyOpenSci community!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-april-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+## <i class="fa-solid fa-heart"></i> To 200 and beyond!
+
+This past month, pyOpenSci reached a major milestone: [200 contributors](https://www.pyopensci.org/our-community/index.html)! We have a diverse and vibrant community of Pythonistas, and are thrilled that 200 (and counting) wonderful people have contributed to pyOpenSci. Will you be next? [Learn more about getting involved](https://www.pyopensci.org/volunteer.html).
+
+
+## <i class="fa-solid fa-camera"></i> Launched: editor spotlight
+
+pyOpenSci editors are volunteers who fulfill a critical role within pyOpenSci. Editors lead the open peer review process for three to four Python packages a year, and also weigh in on group editorial decisions. With everything they do for pyOpenSci, we wanted to celebrate them! In March we kicked off our pyOpenSci editor spotlight, where every Wednesday we share a short interview with one of our editors. You can catch up with previous spotlights, and stay tuned for more, on both our [Fosstodon](https://fosstodon.org/@pyOpenSci) and [LinkedIn](https://www.linkedin.com/company/pyopensci) pages.
+
+## <i class="fa-solid fa-computer"></i> PyCon US 2024 Maintainers Summit
+
+We’ve been talking (a lot) about the [PyCon Maintainers Summit](https://us.pycon.org/2024/events/maintainers-summit/) (which is now full, but [you can sign up for the waitlist](https://us.pycon.org/2024/events/maintainers-summit/)), happening this May in Pittsburgh. The Maintainers Summit, included in your [PyCon registration](https://us.pycon.org/2024/accounts/login/?next=/2024/registration/register), is where we come together to discuss and foster best practices on how to develop sustainable projects and nurture thriving communities. And thanks to [Mariatta Wijaya](https://mariatta.ca/), we’ve got an inside look into everything you can expect in [this incredible video](https://www.youtube.com/watch?v=L-Ok_89QJOM) featuring Kara Sowles, Inessa Pawson, and our Executive Director, Leah Wasser!
+
+<a href="https://www.youtube.com/watch?v=L-Ok_89QJOM" target="_blank">
+ <img src="/images/newsletter/04-2024-newsletter/ms-pycon-2024.png" alt="Watch the video" width="720" height="540" border="10" />
+</a>
+
+## <i class="fa-regular fa-map"></i> Welcome, EOmaps!
+<figure>
```

Comment:
it shouldn't be an issue with the updated video embed. let me know if that's not the case!

Reply:
ok i totally see that now. i think i jumped to a conclusion that the fork / branch was not synced but looking at CI i see the images there. i'm actually not sure why the images aren't rendering in CI. 

let's do this. when you submit another one of these let's both check CI to ensure that images render. if they dont, but render locally on the branch and the branch is synced/ rebased i will open an issue!! i'm not sure if this is a fluke, a workflow weirdness situation because this was in another pr OR just a bug in ci!